backend/alembic/versions/731e18130e01_questions_quiz_relationship.py

```
@@ -0,0 +1,50 @@
+# revision identifiers, used by Alembic.
+revision: str = '731e18130e01'
+down_revision: Union[str, Sequence[str], None] = 'e2dc8f08eb61'
+branch_labels: Union[str, Sequence[str], None] = None
+depends_on: Union[str, Sequence[str], None] = None
+
+
+def upgrade() -> None:
+    """Upgrade schema."""
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.create_table('questions',
+    sa.Column('id', sa.Uuid(), nullable=False),
```

Flake8 (7.2.0)

[error] continuation line under-indented for visual indent

(E128)